### ISSUE TITLE
1.13 [ERROR] WorldGuard incompatibility

### ISSUE THREAD

**Issue by Baktus79** [NoChanceSD/PvPManager-Issues#96]
```
[13:58:26 ERROR]: Error occurred while enabling PvPManager v3.7.6 (Is it up to date?)
java.lang.NoSuchMethodError: com.sk89q.worldguard.bukkit.WorldGuardPlugin.getRegionContainer()Lcom/sk89q/worldguard/bukkit/RegionContainer;
        at nc.bk.<init>(SourceFile:25) ~[?:?]
        at nc.bu.<init>(SourceFile:3060) ~[?:?]
        at me.NoChance.PvPManager.PvPManager.onEnable(SourceFile:49) ~[?:?]
        at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:264) ~[patched_1.13.jar:git-Paper-70]
        at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:325) ~[patched_1.13.jar:git-Paper-70]
        at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:405) ~[patched_1.13.jar:git-Paper-70]
        at org.bukkit.craftbukkit.v1_13_R1.CraftServer.enablePlugin(CraftServer.java:432) ~[patched_1.13.jar:git-Paper-70]
        at org.bukkit.craftbukkit.v1_13_R1.CraftServer.enablePlugins(CraftServer.java:346) ~[patched_1.13.jar:git-Paper-70]
        at net.minecraft.server.v1_13_R1.MinecraftServer.m(MinecraftServer.java:589) ~[patched_1.13.jar:git-Paper-70]
        at net.minecraft.server.v1_13_R1.MinecraftServer.g_(MinecraftServer.java:551) ~[patched_1.13.jar:git-Paper-70]
        at net.minecraft.server.v1_13_R1.MinecraftServer.a(MinecraftServer.java:428) ~[patched_1.13.jar:git-Paper-70]
        at net.minecraft.server.v1_13_R1.DedicatedServer.init(DedicatedServer.java:305) ~[patched_1.13.jar:git-Paper-70]
        at net.minecraft.server.v1_13_R1.MinecraftServer.run(MinecraftServer.java:764) ~[patched_1.13.jar:git-Paper-70]
        at java.lang.Thread.run(Thread.java:748) [?:1.8.0_171]
```

PvPManager breaks with the new WorldGuard plugin version 7.0.0.

**Comment by Tazkee** [comment]
I'm having the same problem, help please

**Comment by NoChanceSD** [comment]
Should be fixed on PvPManager 3.8.1
